po/it.po

```diff
@@ -23440,6 +23452,13 @@ msgid ""
 "For more details, please read the information at the links above\n"
 "\n"
 msgstr ""
+"La prima cosa che devi fare è creare un nuovo Albero Genealogico. Per creare "
```

Comment:
albergo genealogico should be lowercase, the Camel Case Writing Is English Only (and not always).